pkg/agent/core/ngt/service/ngt.go

```diff
@@ -1063,6 +1063,8 @@ func (n *ngt) SaveIndex(ctx context.Context) (err error) {
 		}
 	}()
 	if !n.inMem {
+		log.Info("save index operation started")
+		defer log.Infof("save index operation finished")
```

Comment:
I think it should be implemented in n.saveIndex function

Reply:
Thank you for your comment :pray:
I have fixed: https://github.com/vdaas/vald/pull/2048/commits/59a2f3763f17da868c5091e25a5c52bff0b7d32a

Reply:
Thank you. I have fixed: https://github.com/vdaas/vald/pull/2048/commits/334466731aa7c1a589e0e71ddf8ff9b1b8df08be